week3/src/components/Result.jsx

```diff
@@ -0,0 +1,118 @@
+import { React, useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+
+import styled from "styled-components";
+import 왕관 from "../img/crown.png";
+import Toast from "./Toast.jsx";
+
+function ResultPage({winner}) {
+  const [toastStatus, setToastStatus] = useState(false);
+  
+  function handleShareButton() {
+    setToastStatus(true);
+    const text = document.createElement("textarea");
+    document.body.appendChild(text);
+    text.value = "https://idealwordcup.vercel.app/";
+    text.select();
+    document.execCommand("copy");
+    document.body.removeChild(text);
+  }
+
+  useEffect(() => {
+    if (toastStatus) {
+      setTimeout(() => {
+        setToastStatus(false);
+      }, 1000);
+    }
+  }, [toastStatus]);
+
+  return (
+    <ResultWrapper>
+      <img src={왕관} alt="왕관" />
+      <ImageWrapper>
+        <img src={winner[0].src} alt={winner[0].name} className="item__img" />
+      </ImageWrapper>
+      <strong className="winner__name">{winner[0].name}</strong>
+      <div className="main__buttons">
+        <Link to="/">
+          <button type="button">다시하기</button>
+        </Link>
+        <button type="button" onClick={handleShareButton}>
+          공유하기
+        </button>
+      </div>
+      {toastStatus && <Toast message="📨 링크가 공유되었습니다 !" />}
+
+    </ResultWrapper>
+  );
+}
+
+const ImageWrapper = styled.div`
+  display: flex;
+  margin: 0 auto;
+  gap: 5px;
+
+  img {
```

Comment:
아래처럼

```suggestion
  & > img {
```

로 명시하면 더 명확해지겠다!